SU-SWS/decanter
# Merge 382 namespacing with master

# READY FOR REVIEW

# Summary
- This performs a merge with master and adds it back to the `382-namespacing` branch,  essentially bringing it up to date with all the amazing work accomplished thus far.
- It removes some of the variable name changes, but I'll add them back in next.

# Needed By (Date)
- Soon. 

# Urgency
- We want to get this done before we get any more merge conflicts! 😆 

# Steps to Test

1. Check out this branch
1. Verify the code looks okay.
2. Verify Decanter still works- in particular the main nav stuff

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- JIRA ticket - https://stanfordits.atlassian.net/browse/THEME-14
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
